### ISSUE TITLE
Idea: Comments on Reports

### ISSUE THREAD

**Issue by ThisIsMissEm** [tootsuite/mastodon#6967]
Presently the only way to leave information on a report is to go to the admin page for the user profile, and then that's not visible from the report page. This is a proposal to add comments to reports, something like:

```
report_comments

id bigint
report_id bigint
comment text
account_id bigint
timestamps
```

Having the ability to have many comments on a report would enable discussion about reports in mastodon between moderators. On a similar note, I'd suggest adding an "assignment" or "status" columns to reports, such that we could as a moderator say "I'm dealing with this report" or "status" as a text field such as "contacted user" etc.

I'd be happy to implement this, as it'd greatly help with the moderation duties on switter.at

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Gargron** [comment]
That sounds good to me - would prefer assignment over "status" because "status" is used to describe toots in the code

**Comment by Gargron** [comment]
"state" also works though

**Comment by ThisIsMissEm** [comment]
Really "status" it's a top-level comment; So, maybe "moderator_note" and "assigned_moderator"

**Comment by ThisIsMissEm** [comment]
@Gargron have started an implementation in #7000 — it probably needs some work before merging, but seems like an okay start.

**Comment by ThisIsMissEm** [comment]
This has landed in #7000
